app/web/src/components/AttributesPanel/AttributesPanelItem.vue

```diff
@@ -0,0 +1,1009 @@
+<template>
```

Comment:
this is where the bulk of magic happens...

I (probably) need to refactor this to pull out each input type into their own components (similar to what the "widget" stuff was doing) but it's much easier to style consistently and figure out how it works when it's all together.

Note that I used regular css here because it was just way too hard to follow what was going on otherwise. Also some visual things really needed CSS rules as it had to do with siblings and parents...

We can refactor this back into tailwind classes at some point if it becomes problematic.